onnxruntime/core/providers/cpu/ml/svmclassifier.cc

```diff
@@ -75,6 +75,31 @@ SVMClassifier<T>::SVMClassifier(const OpKernelInfo& info)
   }
 }
 
+template <typename LabelType>
+int _set_score_svm(Tensor* Y, const float& maxweight, const int64_t& maxclass, const int64_t& n,
```

Comment:
>const float& maxweight, const int64_t& maxclass, const int64_t& n [](start = 30, length = 65)

Passing primitive by const reference is not necessary nor performant. At least two instructions instead of 1.

Reply:
fixed